toolchain-fitnesse-plugin-test/wiki/FitNesseRoot/FrontPage/content.txt

```diff
@@ -0,0 +1,15 @@
+!1 Welcome to [[FitNesse][FitNesse.FitNesse]]!
+
+!4 TestSuiteDemo (Praegus demo)
```

Comment:
Wat zit er in die praegus demo dat ook de plugin test?

Reply:
gebruikt om te testen of het project werkte, maar sinds er nu test's in het regressietest suite zit overbodig, weggehaald.